presto/eval.py

```diff
@@ -278,7 +281,7 @@ def evaluate(
         pretrained_model: Optional[PrestoFineTuningModel] = None,
     ) -> Dict:
 
-        test_ds = self.ds_class(self.test_df, target_function=self.target_function)
+        test_ds = self.ds_class(self.test_df, target_function=self.target_function, augment=False)
```

Comment:
you mean no augmentation for inference? This is also the default I guess?

Reply:
right. removed this parameter 8d7e4c11876356a360f693805c6c0444c3fcd09d